packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -75,7 +75,8 @@ export type SuiteAction =
           payload: Partial<AutodetectSettings>;
       }
     | { type: typeof deviceActions.requestDeviceReconnect.type }
-    | { type: typeof SUITE.SET_EXPERIMENTAL_FEATURES; payload?: ExperimentalFeature[] };
+    | { type: typeof SUITE.SET_EXPERIMENTAL_FEATURES; payload?: ExperimentalFeature[] }
+    | { type: typeof SUITE.SET_SIDEBAR_WIDTH; payload: number };
```

Comment:
When we add new data to payload we don't have to refactor all old usages if we use payload as an object.

Reply:
Right. Just this is not applicable for this specific case I am adding here because it doesn't make sense to provide anything else than a number for sidebar width. If we added more data to this action in the future, we should also adjust the action's name to reflect its purpose.

Reply:
OK, it doesn't matter much here.

Reply:
I recognize this will probably never change (I don't see situation when we want to add more fields), but I like consistency and just for this sake of consistency I would prefer to have it wrapped in `{}`,